ui/src/routes/_layout/organizations/$orgId.products.$productId.repositories.$repoId.create-run.tsx

```diff
@@ -89,7 +89,15 @@ const CreateRunPage = () => {
       repositoryId: Number.parseInt(params.repoId),
       requestBody: {
         revision: values.revision,
-        jobConfigs: {},
+        jobConfigs: {
```

Comment:
Commit message typo in "execpt".

Reply:
@Etsija, could running esp. the scanner by default be a problem for us, e.g. performance-wise?

Reply:
I added that commit mainly for testing, but in the end I actually didn't find the time to test it. I could also drop it, I assume you will improve the form to trigger an ORT run anyway and make it configurable via UI which workers are executed?

Reply:
Yes, I'd much rather keep the ORT run fully configurable in the UI, and am currently implementing the form to do so. 

One suggestion for improvement: I can see that currently, not running eg. scanner worker is triggered by not passing any configuration object to it in jobConfigs. This is a bit tricky in the UI sense, as I'm envisioning there would be checkboxes for all workers, and by checking a worker -> it is added to the ORT run.

My suggestion is that in jobConfig, there would be a boolean variable for each worker (default: false), which would determine whether that worker is included in the run or not.  This way, the user can in the UI still configure all workers, but just choose some of them to be run as part of ORT run. This would greatly help in debugging.

Reply:
jobConfigs would then look something like:
```
{
  "analyzer": {
    "includeInRun": true,
    "allowDynamicVersions": true,
    "packageCurationProviders": [],
    "skipExcluded": true
  },
  "scanner": {
    "includeInRun": false,
    "skipExcluded": true
  },
  "reporter": {
    "includeInRun": true,
    "formats": [
      "ortresult"
    ],
    "packageConfigurationProviders": [],
    "assetFiles": [],
    "assetDirectories": []
  },
  "parameters": {}
}
```

Reply:
But all in all, I would drop this commit from the PR, as I am currently editing exactly the same file and putting in the logic to fill out those jobConfigs for different phases.

Reply:
> One suggestion for improvement: I can see that currently, not running eg. scanner worker is triggered by not passing any configuration object to it in jobConfigs. This is a bit tricky in the UI sense, as I'm envisioning there would be checkboxes for all workers, and by checking a worker -> it is added to the ORT run. With current implementation I need to ensure that when a worker is excluded from the run, the corresponding part of jobConfigs is completely deleted.

Please feel free to create an issue for that request, but I'm not sure if we want to change that. The current approach might be inconvenient in the frontend code, but it was no problem to handle that in our internal parameter validation script and it was convenient when manually creating requests. But anyway, we will make an overall review of the API also with your input, so it's helpful to create tickets for everything you would like to have changed.

> But all in all, I would drop this commit from the PR, as I am currently editing exactly the same file and putting in the logic to fill out those jobConfigs for different phases.

Sure, I will drop it, but I'll wait with editing this PR until someone can confirm that it fixes #174.

Reply:
I'm trying to verify the PR, however have some difficulties actually saving the file; back-end logs show that the request goes through OK.

Reply:
I have verified that the download at least works correctly with a REST client, I will look into the cookie topic @mmurto  mentioned below.